legend-engine-application-query/src/main/java/org/finos/legend/engine/application/query/model/DataCubeQuery.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.application.query.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DataCubeQuery
+{
+    public String id;
+    public String name;
+    public String description;
+
+    public Map<String, ?> query;
```

Comment:
shouldnt these be modeled. 
The only thing i would argue we may not want to model is the actual grid config for data cube

Reply:
Do we need this shape here for any reason? I don't think we do, else, we need to move `DataCubeQuery` shape into a common module (out of repl-data-cube). That's work unnecessary at the moment that we can do anytime when the need is there.